services/provider/server/server.go

```diff
@@ -969,6 +986,223 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 
 }
 
+// SHIM for GetStorageClaimConfig to return response to 4.18 clients
+func (s *OCSProviderServer) proxyGetStorageClaimConfigBlock(
```

Comment:
- What is exactly "proxied" here? I don't see a reason to have proxy in the name of this function
- This is not a function for any block storage claim. It is for exactly the storage claim that was the default storage claim in 4.18. Please rename the function accordingly 

```suggestion
func (s *OCSProviderServer) get4_18DefaultBlockStorageClaimConfig(
```